### PR TITLE
[FEAT] - add popup

### PR DESCRIPTION
**Pull request by vlad-lisnenko**

### CODE REVIEW

File: scripts/firebase_script.js [hunk 1]
```diff
@@ -51,6 +51,9 @@ function saveUser() {
         .catch((error) => {
             console.error("Error saving user: ", error);
         });
+
+    OVERLAY.style.display = 'none';
```
Comment (scratchw0w) [inline scripts/firebase_script.js:55]: decide what kind of brackets will we use all the way.

"" or '' and use it everywhere

File: consultation.html [hunk 2]
```diff
@@ -3,9 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <link rel="icon" type="image/x-icon" href="src/computer-logo.png"/>
-    <link rel="stylesheet" href="styles/style.css"/>
-    <link rel="stylesheet" href="styles/course_style.css"/>
+    <link rel="icon" type="image/x-icon" href="../src/computer-logo.png"/>
```
Comment (scratchw0w) [inline consultation.html:6]: I guess we do not need to exit current dir "../"

File: consultation.html [hunk 3]
```diff
@@ -68,7 +68,7 @@
 <div class="ed-container" style="max-height: 800px; justify-content: space-between">
     <div class="ed-content" style="flex: 0">
         <div class="ed-left">
-            <img src="src/consult_photo.jpg" alt="Photo">
+            <img src="../src/consult_photo.jpg" alt="Photo">
```
Comment (scratchw0w) [inline consultation.html:71]: and here

File: index.html [hunk 4]
```diff
@@ -73,6 +73,35 @@
     </div>
 </div>
 
+
+<div id="myModal" class="modal">
```
Comment (scratchw0w) [inline index.html:77]: what is modal?

do you mean model? anyway it is bad naming. never use words like "my" etc. Consider better naming, like attracted_usr_model or usr_model, also I guess ids should not be camelCased, but _  (ex. not myModel but my_model)

File: index.html [hunk 5]
```diff
@@ -73,6 +73,35 @@
...
+                    <option value="CONSULTATION" data-i18n="header_consultation"></option>
+                    <option value="MENTORING" data-i18n="header_mentorship"></option>
+                </select><br><br>
+            </div>
+            <button type="button" onclick="saveUser()">Save User</button>
+        </form>
+    </div>
+</div>
+
+<div id="overlay" class="overlay"></div>
```
Comment (scratchw0w) [inline index.html:103]: is it used for background?

File: styles/style.css [hunk 6]
```diff
@@ -106,6 +107,121 @@ html {
 
 .links a:hover {
     text-decoration: underline;
+
+}
+
+
+/* Modal window*/
```
Comment (scratchw0w) [inline styles/style.css:114]: * Modal

File: styles/style.css [hunk 7]
```diff
@@ -106,6 +107,121 @@ html {
...
+.form-group {
+    margin-bottom: 15px;
+}
+
+.modal label {
+    display: block;
+    margin-bottom: 5px;
+}
+
+.modal input[type="text"],
```
Comment (scratchw0w) [inline styles/style.css:158]: what dies type mean, and why do we use it?

File: scripts/main_script.js [hunk 8]
```diff
@@ -1,5 +1,8 @@
 const SCROLLING_CONTAINER = document.querySelector(".scrolling-container");
 const LINKS = document.querySelector(".links");
+const MODAL = document.getElementById("myModal");
+const OVERLAY = document.getElementById("overlay");
+const CLOSE_BUTTON = document.getElementsByClassName("close")[0];
```
Comment (scratchw0w) [inline scripts/main_script.js:5]: why do we have an array here? Imagine we have more close buttons and 0 is another close button

File: scripts/main_script.js [hunk 9]
```diff
@@ -32,3 +35,20 @@ document.addEventListener(EVENT_CLICK, (event) => {
...
+});
+
+CLOSE_BUTTON.onclick = function () {
+  OVERLAY.style.display = 'none';
+  MODAL.style.display = "none";
+};
+
+window.onclick = function (event) {
+  if (event.target === OVERLAY) {
+    OVERLAY.style.display = 'none';
```
Comment (scratchw0w) [inline scripts/main_script.js:51]: '' or ""

File: scripts/main_script.js [hunk 10]
```diff
@@ -32,3 +35,20 @@ document.addEventListener(EVENT_CLICK, (event) => {
...
   }
 });
+
+document.querySelector(".apply-button").addEventListener(EVENT_CLICK, function ()  {
+  OVERLAY.style.display = 'block';
+  MODAL.style.display = "block";
+});
+
+CLOSE_BUTTON.onclick = function () {
+  OVERLAY.style.display = 'none';
```
Comment (scratchw0w) [inline scripts/main_script.js:45]: we have a repeatable part of code, please, do a func and use it everywhere

File: scripts/main_script.js [hunk 11]
```diff
@@ -32,3 +35,20 @@ document.addEventListener(EVENT_CLICK, (event) => {
     LINKS.classList.remove(TOKEN_ACTIVE);
   }
 });
+
+document.querySelector(".apply-button").addEventListener(EVENT_CLICK, function ()  {
+  OVERLAY.style.display = 'block';
+  MODAL.style.display = "block";
```
Comment (scratchw0w) [inline scripts/main_script.js:41]: i guess here may be a func too

**Review by scratchw0w — CHANGES_REQUESTED** [review]
Please fix comments

File: scripts/firebase_script.js (hunk 1 above)
Comment (vlad-lisnenko) [reply scripts/firebase_script.js:55]: +

File: consultation.html (hunk 2 above)
Comment (vlad-lisnenko) [reply consultation.html:6]: fixed

File: consultation.html (hunk 3 above)
Comment (vlad-lisnenko) [reply consultation.html:71]: fixed

File: index.html (hunk 5 above)
Comment (vlad-lisnenko) [reply index.html:103]: yes)

File: styles/style.css (hunk 6 above)
Comment (vlad-lisnenko) [reply styles/style.css:114]: fixed

File: scripts/main_script.js (hunk 8 above)
Comment (vlad-lisnenko) [reply scripts/main_script.js:5]: fixed

File: scripts/main_script.js (hunk 11 above)
Comment (vlad-lisnenko) [reply scripts/main_script.js:41]: fixed

File: scripts/main_script.js (hunk 10 above)
Comment (vlad-lisnenko) [reply scripts/main_script.js:45]: fixed

File: scripts/main_script.js (hunk 9 above)
Comment (vlad-lisnenko) [reply scripts/main_script.js:51]: fixed

File: styles/course_style.css [hunk 12]
```diff
@@ -138,6 +136,8 @@
 
 .ed-slide img {
     max-width: 100%;
+    height: auto;
+
```
Comment (scratchw0w) [inline styles/course_style.css:140]: remove redundant enter, please

File: styles/course_style.css [hunk 13]
```diff
@@ -338,9 +338,11 @@
         align-items: center;
         font-size: 1.2rem;
         text-align: center;
+        margin-top: 0;
     }
 
     .ed-bottom p {
+
```
Comment (scratchw0w) [inline styles/course_style.css:345]: and here

File: styles/style.css [hunk 14]
```diff
@@ -106,6 +107,119 @@ html {
...
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: rgba(0, 0, 0, 0.7);
+    z-index: 999;
+    width: 80%;
+    max-width: 500px;
+    padding: 20px;
+    border-radius: 5px;
+    border: 1px solid #555555;
+
```
Comment (scratchw0w) [inline styles/style.css:140]: and here

**Review by scratchw0w — CHANGES_REQUESTED** [review]
Please make additional minor change

**Review by scratchw0w — APPROVED** [review]
Great!

**Pull request merged**